pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>au.gov.nla</groupId>
   <artifactId>dl-models</artifactId>
   <packaging>jar</packaging>
-  <version>1.6.16-RELEASE</version>
+  <version>1.6.17-SNAPSHOT</version>
```

Comment:
Wasn't `master` at 1.6.15-SNAPSHOT before? How did it jump to 1.6.16-RELEASE before you merged this in?

Reply:
1.6.15 was released last year https://github.com/nla/dl-models/releases/tag/1.6.15. I guess master was forgotten to update.